changelogs/head.asciidoc

```diff
@@ -32,6 +33,7 @@ https://github.com/elastic/apm-server/compare/7.15\...master[View commits]
 - HTTP server errors (e.g. TLS handshake errors) are now logged {pull}6141[6141]
 - Span documents now duplicate extended HTTP fields, which were previously only under `span.http.*`, under `http.*` {pull}6147[6147]
 - We now record the direct network peer for incoming requests as `source.ip` and `source.port`; origin IP is recorded in `client.ip` {pull}6152[6152]
+- We now collect span destination metrics for transactions with too many spans (for example due to transaction_max_spans or exit_span_min_duration) when collected and sent by APM agents {pull}6200[6200]
```

Comment:
I added 2 entries, but I can keep maybe this one only, I think they're both relevant, but maybe it's too much for this change.

Reply:
Personally I don't find much value in the "Intake API Changes" section, but I don't have a problem with it either. What you have added is fine.

Reply:
The Intake API changes are mainly ment to provide value to agent developers as reference which apm-server version supports which intake fields.